src/Microsoft.Azure.WebJobs.Host/Extensions/JobHostMetadataProvider.cs

```diff
@@ -40,6 +40,8 @@ internal void Init(IBindingProvider root)
             {
                 converter.AddAssemblies((type) => this.AddAssembly(type));
             }
+
+            AddTypesFromGraph(root as IRuleProvider);
```

Comment:
It was for BOT - it came up when I was doing private testing for these changes with the BOT extension. 

---
In reply to: [117296439](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117296439) [](ancestors = 117296439)